### PR TITLE
COOKIE-001 Implement i18n translation 

### PR DESCRIPTION
**Pull request by CiastowskiMichal**

### CODE REVIEW

**Comment by Stared55** [comment]
update PR title with the issue name

File: i18n.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+// (tip move them in a JSON file and import them,
+// or even better, manage them separated from your code: https://react.i18next.com/guides/multiple-translation-files)
+const resources = {
+  en: {
+    translation: {
+      Title: "Goals",
+      Delete: "Delete",
+      AddNewGoal: "Add New Goal",
+      ChangeLanguage: "Change Language"
+    }
```
Comment (Stared55) [inline i18n.js:14]: extract to separeted file `src/locales/en.ts`

File: i18n.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  },
+  pl: {
+    translation: {
+      Title: "Cele",
+      Delete: "Usuń",
+      AddNewGoal: "Dodaj nowy cel",
+      ChangeLanguage: "Zmień język"
+
+    }
```
Comment (Stared55) [inline i18n.js:23]: same

File: i18n.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import i18n from "i18next";
+import { initReactI18next } from "react-i18next";
+
+// the translations
+// (tip move them in a JSON file and import them,
+// or even better, manage them separated from your code: https://react.i18next.com/guides/multiple-translation-files)
+const resources = {
```
Comment (Stared55) [inline i18n.js:7]: move whole file to src/locales/i18n.ts

File: src/components/Item/Item.tsx [hunk 4]
```diff
@@ -5,18 +5,21 @@ import {
...
 
+    const [t, i18n] = useTranslation();
+
     const deleteButton = () => {
         props.onDeleteGoal(props.item);
     }
     return (
         <View style={item.item}>
-            <Text style={item.title}>{props.item.name}</Text><ClearButton title="Delete!" onPress={deleteButton} />
+            <Text style={item.title}>{props.item.name}</Text><ClearButton title={t('Delete')} onPress={deleteButton} />
```
Comment (Stared55) [inline src/components/Item/Item.tsx:22]: keys should be lowercase

File: src/components/MainScreen/MainScreen.tsx [hunk 5]
```diff
@@ -16,6 +20,8 @@ interface Aa {
 
 const MainScreen = () => {
```
Comment (Stared55) [inline src/components/MainScreen/MainScreen.tsx:21]: move to screens module

File: src/components/Item/Item.tsx [hunk 6]
```diff
@@ -5,18 +5,21 @@ import {
...
 import item from './Item.styles';
 import MainScreen from "../MainScreen/MainScreen"
+import { useTranslation } from 'react-i18next';
 
 interface ItemProps { item: any, onDeleteGoal : (goalId : string) => void }
 
 
 const Item = (props: ItemProps) => {
 
+    const [t, i18n] = useTranslation();
```
Comment (Stared55) [inline src/components/Item/Item.tsx:15]: why [] instead of {} ?

Comment (CiastowskiMichal) [reply src/components/Item/Item.tsx]: updated

File: src/components/MainScreen/MainScreen.tsx (hunk 5 above)
Comment (CiastowskiMichal) [reply src/components/MainScreen/MainScreen.tsx:20]: I'll do this on rn-001

File: src/components/Item/Item.tsx (hunk 4 above)
Comment (CiastowskiMichal) [reply src/components/Item/Item.tsx]: updated

File: i18n.js (hunk 3 above)
Comment (CiastowskiMichal) [reply i18n.js]: updated

File: i18n.js (hunk 2 above)
Comment (CiastowskiMichal) [reply i18n.js]: updated

File: i18n.js (hunk 1 above)
Comment (CiastowskiMichal) [reply i18n.js]: updated

**Review by Stared55 — APPROVED** [review]

File: src/locales/i18n.ts [hunk 7]
```diff
@@ -0,0 +1,27 @@
+import i18n from "i18next";
+import { initReactI18next } from "react-i18next";
+import pl from './en';
```
Comment (wikson13) [inline src/locales/i18n.ts:3]: you got names mixed up `pl`-> `./en`

Comment (CiastowskiMichal) [reply src/locales/i18n.ts:3]: updated

**Pull request merged**